rpc/chain/tagger/tagger.go

```diff
@@ -0,0 +1,13 @@
+package tagger
+
+type Tagger interface {
+	Tag() string
+	SetTag(tag string)
+	GroupTag() string
+	SetGroupTag(tag string)
+	DeepCopyTag() Tagger
+}
+
+func DeepCopyTagger(t Tagger) Tagger {
+	return t.DeepCopyTag()
+}
```

Comment:
What's the point of such function? Can't we use `t.DeepCopyTag()` directory? 🤔

Reply:
God knows, I limited myself to moving this code to a separate package so we wouldn't have the mock package depending on the real package.
I'm also quite sure I'll end up getting rid of this, this tagger (and its use in the context of the limiter) only made sense when we wanted to restrict how many requests we allow an account to make when fetching past history, which is as a whole going away.